pkg/auth/auth.go

```diff
@@ -29,13 +29,13 @@ import (
 )
 
 // Authenticator is an interface to represent an external
-// source that should be authenticated against
+// source that should be authenticated against.
 type Authenticator interface {
 	// Auth is used to authenticate to an external service
 	Auth(*api.Client, cfg.Config) (string, error)
 }
 
-// authenticate authenticates!
+// authenticate authenticates with Vault.
```

Comment:
Suggestion: "Authenticate authenticates to Vault, returns true if successful."

Reply:
Done!